helm/aladdin-demo/templates/aladdin-demo-redis.deploy.yaml

```diff
@@ -0,0 +1,23 @@
+{{if .Values.redis.createRedis }}
```

Comment:
Maybe you copied this from another project, but `redis.createRedis` seems a little verbose. Maybe `redis.create` or just `createRedis` is best